### ISSUE TITLE
 FakeYouError: The FakeYouAPI is not available

### ISSUE THREAD

**Issue by ftk789** [leunamcrack/fakeyou.js#2]
You think using fakeyou.js in pure JS for HTML Would cause this? Cause it's made for NodeJS and maybe CORS headers would affect or something...

**Comment by DanLop618** [comment]
I'm actually using it on Node.js and this issue is affecting me too. Hope they can fix it soon

**Comment by ftk789** [comment]
When requesting with pure JS, It gives a CORS error, But the error handler on the wrapper just responds with an API error, So the issue with CORS, But in NodeJS, I should work perfectly fine, For me.

**Comment by mchernoff** [comment]
The error is being thrown from [here](https://github.com/leunamcrack/fakeyou.js/blob/e73c3e227a9b05f723d2a34ba9e1118aab8483a2/src/util/Requester.js#L85).

You can see there's no try/catch around this async block so node gets a fatal error if something is thrown inside of ```__getData```

**Comment by ftk789** [comment]
That's true, I guess the node version can define if the error is there or not, For me, I can use the npm package just fine, With no changes to the actual code structure, Let me know if you're still facing this problem maybe I could help in any shape or form.
